autoinstrumentation/python/Dockerfile

```diff
@@ -14,7 +14,9 @@ WORKDIR /operator-build
 
 ADD requirements.txt .
 
-RUN mkdir workspace && pip install --target workspace -r requirements.txt
+RUN mkdir workspace && apk update && apk add python3-dev gcc libc-dev linux-headers && pip install --target workspace -r requirements.txt
```

Comment:
1. Do we really need this part `apk update && apk add python3-dev gcc libc-dev linux-headers`?
2. Split it to two commands if it's indeed necessary. It will be more layer efficient